collagraph/renderers/pyside/__init__.py

```diff
@@ -0,0 +1,110 @@
+from functools import lru_cache
```

Comment:
I would suggest restructuring the folders/modules a little bit, I think it could be clearer.

* `collagraph/renderers/pyside_renderer.py`
* `collagraph/renderers/pyside/__init__.py`
* `collagraph/renderers/pyside/utils.py` (move code from `__init__.py` here)
* `collagraph/renderers/pyside/widgets/__init__.py`
* `collagraph/renderers/pyside/widgets/action.py`
* `collagraph/renderers/pyside/widgets/<etc>`

I think the fact that so much code is in `__init__.py` now is a little poor in terms of discoverability for maintainers/developers such as you and me.